timely_beliefs/sensors/func_store/knowledge_horizons.py

```diff
@@ -1,25 +1,40 @@
-"""Function store for computing knowledge horizons given a certain event start.
-When passed an event_start = None, these functions return bounds on the knowledge horizon,
+"""Function store for computing knowledge horizons given a certain event start and resolution.
+When passed an event_start = None, these functions return bounds on the knowledge horizon,  # todo: how do the bounds change when event_resolution != sensor.event_resolution
 i.e. a duration window in which the knowledge horizon must lie (e.g. between 0 and 2 days before the event start)."""
 from typing import Optional, Tuple, Union
 from datetime import datetime, timedelta
 
 from timely_beliefs.sensors.utils import datetime_x_days_ago_at_y_oclock
 
 
+def timedelta_for_end_of_event(
+    event_start: Optional[datetime], event_resolution: timedelta
+) -> Union[timedelta, Tuple[timedelta, timedelta]]:
+    """Knowledge horizon always matches the negative event_resolution.
+    This knowledge horizon is anchored to the event_end.
```

Comment:
What does it mean if two timedeltas come back?

Reply:
I see now the documentation at the top of the module, but developers often don't look there when they check a function. It's still a bit opaque to me what these bounds represent.

Reply:
I made getting bounds a separate function argument, and clarified all docstrings of these knowledge horizon functions.